helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -14,7 +14,7 @@ spec:
       annotations:
         description: '{{`Prometheus can''t communicate with Kubernetes API.`}}'
         opsrecipe: prometheus-cant-communicate/
-      expr: rate(prometheus_sd_kubernetes_http_request_total{app!="promxy-app-unique", status_code="<error>"}[15m]) > 0.25
+      expr: rate(prometheus_sd_kubernetes_http_request_total{app!="promxy-app", status_code="<error>"}[15m]) > 0.25
```

Comment:
Also dropping the `unique` suffix from `promxy` after checking on `gauss` that the promxy app no longer has the suffix. Pinging @giantswarm/team-atlas so you're aware